### PR TITLE
[feature #156577846] Create departments and add checks to departments

### PR DESCRIPTION
**Pull request by hariclerry**
#### What does this PR do?
Ensure that Teams can view all jobs in the team account dashboard listed together as one view

#### Description of Tasks to be complete
Create different departments, add checks to departments and filter departments so that when teams log in to the dashboard they should be able to view the jobs per department.

#### How should this be manually tested?
1: Install the project and install the requirements.txt
2: Start the server by running; python manage.py runserver, open localhost  http://localhost:8080 in the browser
3: Login into the site
4: From the dashboard click on the Departments link
5: On the department page create department and save it
6: Go to the create checks links, add the check name and department
7: Click on the Department dropdown menu and select department
8: You should be able to view checks for that particular department.
#### What are the relevant pivotal tracker stories?
#156577846
#### Checklist: I completed these to help reviewers
- [x] My IDE is configured to follow the PEP8 Conventions

No? Unsure? -> configure your IDE, format the code and add the changes with git add . && git commit --amend
- [x] I have added tests to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

No? -> write tests and add them to this commit git add . && git commit --amend

- [x] All new and existing tests passed.

No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is based on the latest changes of the develop branch.

No? Unsure? -> execute command git pull --rebase origin develop

### CODE REVIEW

**Comment by hariclerry** [comment]
Hello Team, please review my code and let me know what you guys think.

**Review by dondrzzy — CHANGES_REQUESTED** [review]
I have run your branch and it looks good. Nice work. These are my UX and UI reviews
1: It would be better to place the departments link on the dashboard as opposed to the drop downlink that usually holds links to logout and account_settings
2:The filter on the failed checks view does not work
3: The placement of the edit department is also off. Its under name. The <th>name</th> should contain a name. It might be better to add it the department to the table columns(td).
4: The departments filter on the checks page would in my view look better if you float it to the right. Or just to the right of the `failed_checks` button and and of the same size maybe to provide more sync. In my view.

**Comment by hariclerry** [comment]
@dondrzzy, great that you pointed those things out, I will work on them. But for the placement of the edit department button, am following health checks UI design.

**Comment by hariclerry** [comment]
And also, the failed checks filter is working on my machine.

**Review by dondrzzy — CHANGES_REQUESTED** [review]
I have run your branch and it looks good. Nice work. These are my UX and UI reviews
1: It would be better to place the departments link on the dashboard as opposed to the drop downlink that usually holds links to logout and account_settings
2:The filter on the failed checks view does not work
3: The placement of the edit department is also off. Its under name. The <th>name</th> should contain a name. It might be better to add it the department to the table columns(td).
4: The departments filter on the checks page would in my view look better if you float it to the right. Or just to the right of the `failed_checks` button and and of the same size maybe to provide more sync. In my view.

File: hc/front/tests/test_add_department.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        self.client.login(username="alice@example.org", password="password")
+
+        response = self.client.get("/departments/add/")
+        self.assertEqual(response.status_code, 200)
+
+    def test_department_can_be_added(self):
+        """ Test that new department can be created using URL """
+        self.client.login(username="alice@example.org", password="password")
+
+        self.client.get("/departments/add/")
```
Comment (aenkya) [inline hc/front/tests/test_add_department.py]: unnecessary line here

Comment (hariclerry) [reply hc/front/tests/test_add_department.py]: Okay. Worked on that.

**Pull request merged**